riotfile.py

```diff
@@ -1809,23 +1809,35 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             },
             venvs=[
                 Venv(
-                    pys=select_pys(min_version="3.7", max_version="3.8"),
-                    pkgs={"urllib3": ["~=1.26.4"]},
+                    # Support added for Python 3.7 in 1.25.0
```

Comment:
We are testing the latest release of 1.25 (not v1.25.0). Will this negatively impact our reporting?

Reply:
Good catch, pinning all specific patch versions where necessary